resources/zib-ContactInformation-E-mailAdresses.xml

```diff
@@ -0,0 +1,75 @@
+    <telecom>
+      <system value="email" />
+      <value value="info@nictiz.nl" />
+      <use value="work" />
+    </telecom>
+  </contact>
+  <description value="Contact information includes a person's telephone numbers en e-mail addresses." />
+  <purpose value="This ContactPoint resource represents the Dutch [zib ('Zorginformatiebouwsteen', i.e. Health and Care Information Model) ContactInformation v1.2(2020)](https://zibs.nl/wiki/ContactInformation-v1.2(2020EN))." />
+  <copyright value="Copyright and related rights waived via CC0, https://creativecommons.org/publicdomain/zero/1.0/. This does not apply to information from third parties, for example a medical terminology system. The implementer alone is responsible for identifying and obtaining any necessary licenses or authorizations to utilize third party IP in connection with the specification or otherwise." />
+  <fhirVersion value="4.0.1" />
+  <mapping>
+    <identity value="zib-contactinformation-v1.2-2020EN" />
+    <uri value="https://zibs.nl/wiki/ContactInformation-v1.2(2020EN)" />
+    <name value="zib ContactInformation-v1.2(2020EN)" />
+  </mapping>
+  <kind value="complex-type" />
+  <abstract value="true" />
+  <type value="ContactPoint" />
+  <baseDefinition value="http://hl7.org/fhir/StructureDefinition/ContactPoint" />
+  <derivation value="constraint" />
+  <differential>
+    <element id="ContactPoint">
+      <path value="ContactPoint" />
+      <short value="EmailAddresses" />
+      <comment value="Implementation of the &quot;EmailAddresses&quot; concept of zib ContactInformation.&#xD;&#xA;&#xD;&#xA;To implement the entire zib, the complementary profile [zib-ContactInformation-TelephoneNumbers](http://nictiz.nl/fhir/StructureDefinition/zib-ContactInformation-TelephoneNumbers) should be used. Both profiles can be added in a slice with the discriminator set to `profile`/`$this`." />
+      <alias value="EmailAdressen" />
+      <mapping>
+        <identity value="zib-contactinformation-v1.2-2020EN" />
+        <map value="NL-CM:20.6.3" />
+        <comment value="EmailAddresses" />
+      </mapping>
+    </element>
+    <element id="ContactPoint.system">
+      <path value="ContactPoint.system" />
+      <short value="email" />
+      <min value="1" />
+      <fixedCode value="email" />
+    </element>
+    <element id="ContactPoint.value">
+      <path value="ContactPoint.value" />
+      <short value="EmailAddress" />
+      <alias value="EmailAdres" />
+      <mapping>
+        <identity value="zib-contactinformation-v1.2-2020EN" />
+        <map value="NL-CM:20.6.7" />
+        <comment value="EmailAddress" />
+      </mapping>
+    </element>
+    <element id="ContactPoint.use">
+      <path value="ContactPoint.use" />
+      <short value="EmailAddressType" />
+      <alias value="EmailSoort" />
+      <binding>
+        <strength value="required" />
+        <description value="Use of contact point for the zib ContactInformation concept EmailAddresses" />

```

Comment:
Zou hier ook niet naar de conceptmap verwezen moeten worden zoals gedaan is bij TelephoneNumbers??